cms/models.py

```diff
@@ -1303,10 +1306,23 @@ class Meta:
 
     parent_page_types = ["CourseIndexPage"]
 
-    content_panels = [
-        FieldPanel("course"),
-        FieldPanel("topics"),
-    ] + ProductPage.content_panels
+    content_panels = (
+        [
+            FieldPanel("course"),
+            FieldPanel("topics"),
+        ]
+        + ProductPage.content_panels
+        + [
+            MultiFieldPanel(
+                [
+                    FieldPanel("course_run", widget=forms.Select),
+                    FieldPanel("price"),
+                ],
+                heading="Change Price",
```

Comment:
Maybe `Add Price` or `Set Price` might be better names since it is for Create/Edit both forms?

Reply:
I have a follow-up question. These fields are of no use for the Create forms. As we only update pricing for already created Courses/Programs.

Reply:
`Set Price` looks good.